nevergrad/functions/images/core.py

```diff
@@ -32,36 +34,54 @@ def __init__(self, problem_name: str = "recovering", index: int = 0) -> None:
         # Storing data necessary for the problem at hand.
         assert problem_name == "recovering"  # For the moment we have only this one.
         assert index == 0  # For the moment only 1 target.
-        #path = os.path.dirname(__file__) + "/headrgb_olivier.png"
+        # path = os.path.dirname(__file__) + "/headrgb_olivier.png"
         path = Path(__file__).with_name("headrgb_olivier.png")
         image = PIL.Image.open(path).resize((self.domain_shape[0], self.domain_shape[1]), PIL.Image.ANTIALIAS)
-        self.data = np.asarray(image)[:,:,:3]  # 4th Channel is pointless here, only 255.
+        self.data = np.asarray(image)[:, :, :3]  # 4th Channel is pointless here, only 255.
 
-        # if problem_name == "adversarial": 
-        #     assert index <= 100  # If we have that many target images.
-        #     self.data = ..... (here we load the imagee correspnding to index and problem_name; this is
-        #         # the attacked image.)
-
-        array = ng.p.Array(init=128 * np.ones(self.domain_shape), mutable_sigma=True,)
+        array = ng.p.Array(init=128 * np.ones(self.domain_shape), mutable_sigma=True, )
         array.set_mutation(sigma=35)
         array.set_bounds(lower=0, upper=255.99, method="clipping", full_range_sampling=True)
         max_size = ng.p.Scalar(lower=1, upper=200).set_integer_casting()
         array.set_recombination(ng.p.mutation.Crossover(axis=(0, 1), max_size=max_size)).set_name("")  # type: ignore
+
+        if problem_name == "adversarial":
+            self.targeted = params["targeted"] if ("targeted" in params) else False
+            self.epsilon = params["epsilon"] if ("epsilon" in params) else 0.05
+            self.image_size = params["image_size"]
+            self.domain_shape = (self.image_size, self.image_size, 3)
+            self.image = params["image"]
+            self.label = params["label"]
+            self.classifier = params["classifier"]
```

Comment:
I don't think anyone will be able to use this if they need to provide a classifier, I agree with Olivier, there should be one provided, or a selection of classifiers to choose from?
Also, if they are big, they must not be submitted within the repo, and need to be hosted somewhere else and downloaded